test/Ansatz/Chain_test.jl

```diff
@@ -208,5 +208,18 @@
         end
     end
 
+    @testset "Normalization" begin
```

Comment:
Don't create a `testset` with only one `testset`. You can leave it like `@testset normalize`.